pdns/pdns_recursor.cc

```diff
@@ -1654,12 +1654,12 @@ static void startDoResolve(void *p)
     if(!dc->d_tcp) {
       struct msghdr msgh;
       struct iovec iov;
-      char cbuf[256];
-      fillMSGHdr(&msgh, &iov, cbuf, 0, (char*)&*packet.begin(), packet.size(), &dc->d_remote);
+      cmsgbuf_aligned cbuf;
+      fillMSGHdr(&msgh, &iov, &cbuf, 0, (char*)&*packet.begin(), packet.size(), &dc->d_remote);
       msgh.msg_control=NULL;
 
       if(g_fromtosockets.count(dc->d_socket)) {
-	addCMsgSrcAddr(&msgh, cbuf, &dc->d_local, 0);
+	      addCMsgSrcAddr(&msgh, &cbuf, &dc->d_local, 0);
```

Comment:
Just a nit but the indentation seems off.

Reply:
will fix